src/internal/views/nunjucks/view-licences/licence.njk

```diff
@@ -28,15 +29,17 @@
             </a>
           </li>
 
-          {% if returns.length %}
+          {% if returns.length > 0 %}
             <li class="govuk-tabs__list-item">
```

Comment:
Same here, should the first LI have `govuk-tabs__list-item--selected` or is it preselected without it?

Reply:
In this case the summary tab is always the selected tab.